README.md

```diff
@@ -31,3 +31,58 @@ cargo build --release
 ../target/release/juice-examples --help
 # which means, you can run the examples from the juice-examples README
 ```
+
+
+### Dependencies
+
+#### Cap'n'Proto
+
+[cpanp is a data interchange format](https://capnproto.org/) that is used to store and load networks with weights for [Juice](https://github.com/spearow/juice/juice).
+
+`capnproto` and `capnproto-libs` plus their development packages are the ones needed from your package manager.
+
+#### Cuda
+
+Getting the cuda libraries up poses to be the first road-block many users face.
+
+To get things working one needs to set the following environment variables:
+
+```zsh
+# examplary paths, unlikely to work for your local setup!
+export CUDNN_INCLUDE_DIR=/opt/cuda/include
+export CUDNN_LIB_DIR=/opt/cuda/targets/x86_64-linux/lib/
+export CUBLAS_INCLUDE_DIR=/opt/cuda/include
+export CUBLAS_LIB_DIR=/opt/cuda/targets/x86_64-linux/lib/
+```
+
+depending on __your local__ installation setup.
+
+The currently supported cuda version is `cuda-10` (details in #114 and #115 )
```

Comment:
```suggestion
The currently supported cuda version is `cuda-10-*` (details in #114 and #115 )
```

We support .0 and .2 I think?

Reply:
Any major of version 10 , since those are guaranteed to have a stable API.